201-alert-to-queue-with-logic-app/azuredeploy.json

```diff
@@ -0,0 +1,442 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "EndpointName": {
+            "type": "string",
+            "metadata": {
+                "description": "Give a unique name that will be used to create the web endpoint for your Logic app. As this is used in a DNS name it must be globally unique."
+            }
+        },
+        "QueueName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the queue you would like to drop the messages dropped into. The queue must already exist before you run the Logic app."
+            }
+        },
+        "StorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the storage account that contains the queue."
+            }
+        },
+        "StorageAccountKey": {
```

Comment:
I think you can get this using the the template function listkeys instead of asking the user to input this 
https://azure.microsoft.com/en-us/documentation/articles/resource-group-template-functions/